sinonjs/samsam
# Security: upgrade dependencies

This PR upgrades a couple of dependencies that have security alerts against them

#### How to verify - mandatory
1. Check out this branch
1. `npm ci`
1. `npm ls cryptiles`
1. Observe that it is not installed
1. `npm ls cache-path-relative`
1. Observe that `v1.0.2` is the only installed version

#### Checklist for author
